_pages/en_US/installing-boot9strap-(kartdlphax).txt

```diff
@@ -49,6 +50,8 @@ On the **target 3DS** (the 3DS that you are trying to modify):
 1. Create a folder named `boot9strap` on the root of your SD card
 1. Copy `boot9strap.firm` and `boot9strap.firm.sha` from the boot9strap `.zip` to the `/boot9strap/` folder on your SD card
 1. Copy `SafeB9SInstaller.bin` from the SafeB9SInstaller `.zip` to the root of your SD card
+1. Copy `usm.bin` from the unSAFE_MODE `.zip` to the root of your SD card
+1. Copy the `slotTool` folder from the unSAFE_MODE `.zip` to the `3ds` folder on your SD card
```

Comment:
people won't have a `3ds` folder at this point and will likely interpret this as `Nintendo 3DS`. So you need a separate instruction to get people to create the `3ds` folder